common/tools/dev-tool/src/util/testProxyUtils.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { spawn } from "child_process";
+import path from "path";
+import { IncomingMessage, request, RequestOptions } from "http";
+import fs from "fs-extra";
+import { createPrinter } from "./printer";
+
+const log = createPrinter("test-proxy");
+export async function startProxyTool(mode: string | undefined) {
+  log.info(`===TEST_MODE="${mode}"===`);
+  log.info(
+    `Attempting to start test proxy at http://localhost:5000 & https://localhost:5001.\n`
```

Comment:
Can these ports be parametrized?

Reply:
Logged an issue for this at #15829, will tackle in a future PR.